hibernate-cache/src/main/java/org/infinispan/hibernate/cache/util/Tombstone.java

```diff
@@ -191,7 +195,9 @@ private ExcludeTombstonesFilter() {}
 
 		@Override
 		public boolean accept(Object key, Object value, Metadata metadata) {
-			return !(value instanceof Tombstone);
+         boolean b = !(value instanceof Tombstone);
+         log.tracef("Is value %s for key %s is tombstone? %b", value, key, b);
```

Comment:
I wouldn't log here...

Reply:
Why not? I added this because I had no way of knowing which of the two entries in the cache that was being returned (instead of expected one entry) was a tombstone...